faster_sam/cloudformation.py

```diff
@@ -49,6 +93,25 @@ def multi_constructor(loader: CFLoader, tag_suffix: str, node: yaml.nodes.Node)
 
 
 def construct_getatt(node: yaml.nodes.Node) -> List[Any]:
+    """
+    Constructs a CloudFormation GetAtt element from a node.
```

Comment:
Se minha sugestão de descrição para a função anterior for aceita acredito valeria a pena manter a mesma linguagem aqui.

_Custom YAML node constructor for AWS CloudFormation GetAtt intrinsic function._